### PR TITLE
Use travis container infastructure

### PR DESCRIPTION
**Pull request by rmcgibbo**

### CODE REVIEW

**Comment by jchodera** [comment]
Cool!

File: .travis.yml [hunk 1]
```diff
@@ -12,18 +31,17 @@ env:
...
-  - sudo pip install nose
+  - export CC=clang
+  - export CXX=clang++
   - export ASAN_SYMBOLIZER_PATH=/usr/bin/llvm-symbolizer-3.3
 
 script:
-  - cmake -DCMAKE_INSTALL_PREFIX=$HOME/OpenMM -DOPENMM_BUILD_STATIC_LIB=$OPENMM_BUILD_STATIC_LIB .
+  - virtualenv --system-site-packages openmm_env
+  - source openmm_env/bin/activate
+  - cmake -DCMAKE_INSTALL_PREFIX=$HOME/OpenMM -DOPENMM_BUILD_STATIC_LIB=$OPENMM_BUILD_STATIC_LIB.
```
Comment (mpharrigan) [inline .travis.yml]: Missing space before .

**Comment by peastman** [comment]
And the tests ran in only about 8 minutes.  Nice!

**Comment by rmcgibbo** [comment]
I think this is good to merge.

**Pull request merged**